portal/static/js/src/components/ExportInstruments.vue

```diff
@@ -89,7 +89,7 @@
                 var self = this;
                 tnthAjax.getInstrumentsList(false, function (data) {
                     if (!data || !data.length) {
-                        document.querySelector("#instrumentsExportErrorMessage").innerText = i18next.t("Error occurred retrieving instruments list");
+                        document.querySelector("#instrumentsExportErrorMessage").innerText = i18next.t("Unable to retrieving instruments list.");
```

Comment:
Sorry for not being more clear- 
I don't think we need to include a reference to the specific API that failed- otherwise we would need different error messages for each AJAX call or API (maybe `tnthAjax` could return a generic error instead?).

This wouldn't normally be a big deal, but with i18n and 10+ languages I'd like to minimize the translation churn.

Since we're already translating this string, could you either use the original (so we don't have to re-translate it), or remove the translation call (`i18next.t()`)?

Reply:
thanks, ivan, ok, think I got it now.  I have it just display the generic error returned from the call.  hope this will take.